dockerfile

```diff
@@ -164,5 +164,10 @@ ENV LIGHTDASH_CONFIG_FILE /usr/app/lightdash.yml
 COPY ./docker/prod-entrypoint.sh /usr/bin/prod-entrypoint.sh
 
 EXPOSE 8080
+
+# Healthcheck for backend service
+HEALTHCHECK --interval=30s --timeout=3s --retries=5 \
```

Comment:
Great suggestion, for reference, we use the following settings for livenessProbe on k8s , should we match them ? :thinking:  https://github.com/lightdash/helm-charts/blob/main/charts/lightdash/values.yaml#L90

```
  initialDelaySeconds: 30
  timeoutSeconds: 60
  periodSeconds: 30
  ```




At least we should use `--start-period` to make sure we don't get a false positive from /health when for example a migration is running. 

Since Kubernetes 1.8, the Docker HEALTHCHECK has been [disabled explicitly](https://github.com/kubernetes/kubernetes/pull/50796) in Kubernetes. so we don't need to make any changes on cloud. This will only affect local development.